src/scss/settings/_variables.scss

```diff
@@ -38,12 +38,16 @@ $type: (
         narrow: (16, 21)
     ),
     large--scaleUp: (
-        default: (28, 36),
+        default: (32, 32),
         narrow: (18, 24)
     ),
     jumbo--scaleUp: (
-        default: (64, 72),
+        default: (44, 48),
         narrow: (28, 32)
+    ),
+    mid--podTitle: (
```

Comment:
Should name this more generically as it may one day be used for things that aren't podTitles :)